janus.c

```diff
@@ -191,16 +191,42 @@ static gchar *local_ip = NULL;
 gchar *janus_get_local_ip(void) {
 	return local_ip;
 }
-static gchar *public_ip = NULL;
+static GPtrArray *public_ip = NULL;
+guint janus_get_public_ip_count(void) {
+	return public_ip ? public_ip->len : 0;
+}
+gchar *janus_get_public_ip_index(guint index) {
```

Comment:
I don't think the name of the method needs to change: if I do `janus_get_public_ip(3)` then it's clear I'm getting a specific one out of many.